admin-dev/themes/new-theme/scss/components/_modal.scss

```
@@ -0,0 +1,63 @@
+/*!
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *
+ */
+
+.modal-iframe {
+  .modal-dialog {
+    width: 90%;
+    max-width: 1200px;
+    max-height: 95%;
+    margin: 0 auto;
+
+    .modal-content {
+      height: 100%;
+      padding: 0;
+      margin: 0;
+
+      .modal-body {
+        padding: 0;
+
+        .modal-iframe-loader {
```

```suggestion
        .#{$component-name}-loader {
```

So we would be able to do this and keep compatible

---

I tried this, but it didn't work and it broke all the integration So I reverted the change because I couldn't find how to make it work